test/integration/clusterimageset-updater.sh

```
@@ -14,9 +17,39 @@ cp -a "${suite_dir}"/* "${workdir}"
+            'phase': 'Accepted',
+            'pullSpec': 'quay.io/openshift-release-dev/ocp-release:4.21.99-multi',
+            'downloadURL': ''
+        }).encode())
+    def log_message(self, *args):
+        pass
+
+http.server.HTTPServer(('127.0.0.1', ${MOCK_PORT}), Handler).serve_forever()
+" &
+MOCK_PID=$!
+# Wait for the server to be ready.
+for i in $(seq 1 20); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)